src/main/scala/chisel3/Driver.scala

```diff
@@ -135,4 +136,8 @@ object Driver extends BackendCompilationUtilities {
   }
 
   def targetDir(): String = { target_dir getOrElse new File(".").getCanonicalPath }
+
+  val version = BuildInfo.version
+  val chiselVersionString = BuildInfo.toString
+  println(chiselVersionString)
```

Comment:
Can we put the print as part of an actual function? And use the same logging / printing infrastructure as everything else? Or better yet, put it in the generated FIRRTL as a comment?